packages/libs/react-ui/src/components/index.ts

```diff
@@ -74,11 +77,18 @@ export { Button } from './Button';
 export { Card } from './Card';
 export { ContentHeader } from './ContentHeader';
 export { Divider } from './Divider/Divider';
+export {
+  Input,
+  InputWrapper,
```

Comment:
Would it make sense to update this component to be named FormFieldWrapper? Right now it feels very tied to the Input. What do you think?

Reply:
yeah you are right! ill do it

Reply:
I think it's better to do that as part of a separate PR to keep things separated. I'll add a ticket to the board and assign you to it!

Reply:
https://app.asana.com/0/1205693948735844/1205783514269055

We can work on this in the next sprint